src/main/scala/org/apache/spark/sql/hive/plan/GenerateDeltaLogCommand.scala

```diff
@@ -21,7 +21,7 @@ case class GenerateDeltaLogCommand(table: Option[CatalogTable],
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val path = if (table.isDefined) {
-      table.get.storage.locationUri.get.getPath
+      table.get.storage.locationUri.get.toString
```

Comment:
why toString here?

Reply:
Yeah this is intresting when you mention oci://nm@bc/a/b/c get path gives /a/b/c but to string only gives you back  "oci://nm@bc/a/b/c"